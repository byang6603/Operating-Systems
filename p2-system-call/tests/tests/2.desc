Simple crash test. Writes crash string, calls crash.
