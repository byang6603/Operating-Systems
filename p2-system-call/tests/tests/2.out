lapicid 0: panic: XV6_TEST_ERROR CRASH SYSTEM CALL INITIATED. CRASHING SYSTEM.
